### PR TITLE
Create form section 

### PR DESCRIPTION
**Pull request by Yacoubou-seidou**
- Complete formsfree end point implementation
- Update README by adding Authors
- Complete HTML and CSS
- Complete the linter test with no error

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -152,6 +152,18 @@ <h5>
...
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+    coding. I’d love to help with it</p>
+    <form class="form_submit" action="https://formspree.io/f/xyyawdao" method="post">
+    <input class="input_element" name="name" placeholder="Full name" type="text" value="Yeremias" required maxlength="30">
+    <input class="input_element" type="email" name="email" placeholder="Email" required value="natajayanj14@gmail.com">
+    <textarea class="text_elemnt" name="message" placeholder="Write your message" id="" cols="30" rows="10" required maxlength="500"></textarea>
+    <button type="submit" class="btn_details submit_btn">Get in touch</button>
+    </form>
+    <hr class="bottom_line">
+</section>
```
Comment (anagudelogu) [inline index.html]: - [ ] You did a great job working on this milestone! 💯 However, please make sure to stick as much as possible to the given Figma design. At the moment, the text for this section is different than the one in the template. Please consider fixing it! 🤓 🔧 

|Template|Your design|
|---|---|
| image | Screenshot from 2023-02-21 15-22-40 |

- [x] Also, please add some padding to your inputs so the text doesn't look like is touching the edges! 🤓 🔧 

- [ ] **[OPTIONAL]:** Is okay if you delete the "value" property of your inputs, they are present in the design mainly to show the color of the font and the spacing of the input. In reality, having these arbitrary "default" values doesn't provide a great user experience! 🤓 🔧

File: style.css [hunk 2]
```diff
@@ -294,4 +294,83 @@ body {
...
   }
+
+  .form_section {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-image: url("/pictures/contactformshapes-mobile@2x.png");
```
Comment (anagudelogu) [inline style.css]: - [ ] **[OPTIONAL]:** Please consider using relative paths over absolute ones, they are often easier to handle! 🤓 🔧 

```suggestion
    background-image: url("pictures/contactformshapes-mobile@2x.png");
```

or

```suggestion
    background-image: url("./pictures/contactformshapes-mobile@2x.png");
```

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi Team! @Yacoubou-seidou @yawsamcode @prius2055, 
I'm Andres.


While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -152,6 +152,18 @@ <h5>
     </ul>
     <hr>
   </section>
+<section class="form_section">
+  <h2>Cantact me</h2>
```
Comment (Shisui6) [inline index.html:156]: Kindly correct the spelling of contact here🙏

**Review by Shisui6 — APPROVED** [review]
Hi @Yacoubou-seidou, @yawsamcode , and @prius2055, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Great design, excellent attention to detail ✅
- HTML validations work ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**